mm2src/coins/lp_coins.rs

```diff
@@ -2098,6 +2104,28 @@ impl MmCoinEnum {
             _ => false,
         }
     }
+
+    fn on_token_deactivated(&self, ticker: &str) -> Result<(), String> {
+        match self {
+            MmCoinEnum::UtxoCoin(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::QtumCoin(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::Qrc20Coin(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::EthCoin(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::Bch(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::SlpToken(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::Tendermint(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::TendermintToken(ref c) => c.on_token_deactivated(ticker),
+            #[cfg(not(target_arch = "wasm32"))]
+            MmCoinEnum::LightningCoin(ref c) => c.on_token_deactivated(ticker),
+            #[cfg(not(target_arch = "wasm32"))]
+            MmCoinEnum::ZCoin(ref c) => c.on_token_deactivated(ticker),
+            MmCoinEnum::Test(_) => Ok(()),
+            #[cfg(all(not(target_os = "ios"), not(target_os = "android"), not(target_arch = "wasm32")))]
+            MmCoinEnum::SolanaCoin(ref c) => c.on_token_deactivated(ticker),
+            #[cfg(all(not(target_os = "ios"), not(target_os = "android"), not(target_arch = "wasm32")))]
+            MmCoinEnum::SplToken(ref c) => c.on_token_deactivated(ticker),
+        }
+    }
```

Comment:
This function can be removed now that `fn on_token_deactivated` is part of `MmCoin` trait